### PR TITLE
Changes to theme background color for Gradient Light and Gradient Dark themes 

### PR DESCRIPTION
**Pull request by samiaab1990**
Modified the background color for Gradient Light and Gradient Dark themes 

### Changes
- Changed background color of Gradient Light to: `background-color: #f9ccff;
background-image: linear-gradient(295deg, #f9ccff 0%, #e6bbf9 11%, #9ec6f9 32%, #55e6ee 60%, #91f5d1 74%, #f9ffbf 98%);` from `background: rgb(255,253,141);
background: linear-gradient(142deg, rgba(255,253,141,1) 0%, rgba(252,183,255,1) 35%, rgba(144,236,255,1) 100%);`

- Changed background color of Gradient Dark to: `background-color: #4e037d;
background-image: linear-gradient(162deg, #4e037d 8%, #40006a 22%, #270075 60%, #032e79 92%);
;` from
`background-color: #280760;
background-image: linear-gradient(225deg, #280760 47%, #03135e 92%);`

### Checklist
- [X] Updated the changelog at `CHANGES.md`- added changes under **Themes:**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -20,10 +20,16 @@ Developer Tools:
...
 
+Themes:
+
+- Modified background color in css for Gradient Light and Gradient Dark themes[Samia Ali][]
+
 [Richard Gibson]: https://github.com/gibson042
 [Bradley Mackey]: https://github.com/bradleymackey
 [Björn Ebbinghaus]: https://github.com/MrEbbinghaus
 [Josh Goebel]: https://github.com/joshgoebel
+[Samia Ali] https://github.com/samiaab1990
```
Comment (joshgoebel) [inline CHANGES.md:31]: Missing `:`

File: CHANGES.md [hunk 2]
```diff
@@ -20,10 +20,16 @@ Developer Tools:
 
 - (chore) add gzip size compression report (#3400) [Bradley Mackey][]
 
+Themes:
+
+- Modified background color in css for Gradient Light and Gradient Dark themes[Samia Ali][]
```
Comment (joshgoebel) [inline CHANGES.md:25]: missing space.

**Comment by joshgoebel** [comment]
Any further context for the "why" of this change?

File: CHANGES.md (hunk 1 above)
Comment (samiaab1990) [reply CHANGES.md]: Resolved

File: CHANGES.md (hunk 2 above)
Comment (samiaab1990) [reply CHANGES.md]: Resolved

File: src/styles/gradient-dark.css [hunk 3]
```diff
@@ -6,8 +6,11 @@ Gradient Dark (c) Samia Ali <samiaab1990@gmail.com>
 
 .hljs
 {
-background: rgb(80,31,122);
-background: linear-gradient(166deg, rgba(80,31,122,1) 0%, rgba(40,32,179,1) 80%);
+display: block;
+overflow-x: auto;
+padding: 0.5em;
```
[the remark this answers is not in the document]
Comment (samiaab1990) [reply src/styles/gradient-dark.css]: Removed

**Comment by samiaab1990** [comment]
> Any further context for the "why" of this change?

Wanted to enhance the background of the original themes: additionally made the color of .hljs-string in the light theme a little darker so it's visible in contrast with the background-just some aesthetic changes I thought would look better.

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@samiaab1990 Thanks!

**Pull request merged**